docs/application-development/testing.md

```diff
@@ -168,7 +186,42 @@ Content is currently unavailable.
 </TabItem>
 <TabItem value="python">
 
-Content is currently unavailable.
+To cancel an Activity in Temporal, you can use the [`WorkflowHandle.cancel()`](https://python.temporal.io/temporalio.client.WorkflowHandle.html#cancel) method, which will send a cancellation request to the running Workflow.
+
+```python
+import asyncio
+import uuid
+from temporalio.client import Client
+from temporalio.worker import Worker
+
+# Import your Workflow definition
+from hello.hello_cancellation import CancellationWorkflow
+
+async def test_cancel_workflow(client: Client):
+    task_queue_name = str(uuid.uuid4())
+    async with Worker(
+        client,
+        task_queue=task_queue_name,
+        workflows=[CancellationWorkflow],
+    ):
+        # Start your Workflow 
+        handle = await client.start_workflow(
+            CancellationWorkflow.run,
+            id=(str(uuid.uuid4())),
+            task_queue=task_queue_name,
+        )
+
+        # Wait for the Activity to start
+        await asyncio.sleep(5)
+
+        # Cancel the workflow
+        await handle.cancel()
+
+        # Assert that the workflow has been canceled
+        assert "CANCELED" == (await handle.describe()).status
+```
+
+The async `cancel()` method sends a cancellation request to the running Workflow. If the Workflow is running an Activity that listens to cancellation, it will terminate the Activity.
```

Comment:
```suggestion
The async `cancel()` method sends a cancellation request to the running Workflow. If the Workflow is running an, it will cancel the Activity.
```